README.md

```diff
@@ -31,33 +50,47 @@ _Note_: These tools are intended for testing and as reference implementation of
 
 ## Getting Started
 
-These instructions will get you a copy of the project up and running on your local machine for development and testing purposes. See Docker for instructions on how to test against network equipment.
+These instructions will get you a copy of the project up and running on your local machine.
 
 ### Prerequisites
 
 Install __go__ in your system https://golang.org/doc/install. Requires golang1.7+.
 
-### Clone
+### Download sources
 
-Clone the project to your __go__ source folder:
 ```
-mkdir -p $GOPATH/src/github.com/google/
-cd $GOPATH/src/github.com/google/
-git clone https://github.com/google/gnxi.git
+go get -v github.com/google/gnxi/...
 ```
 
-### Running
+### Building and installing binaries
+
+```
+cd $GOPATH
+mkdir bin
+go install github.com/google/gnxi/...
+ls -la $GOPATH\bin
+```
 
-To run the binaries:
+### Generating certificates
 
 ```
-cd $GOPATH/src/github.com/google/gnxi/gnmi_get
-go run ./gnmi_get.go
+cd $GOPATH\bin
+./../src/github.com/google/gnxi/certs/generate.sh
 ```
 
-## Docker
+### Running a client
 
-[FAUCET](https://github.com/faucetsdn/gnmi) currently includes a [Dockerfile](https://github.com/faucetsdn/gnmi/blob/master/Dockerfile) to setup the environment that facilitates testing these tools against network equipment.
+```
+cd $GOPATH\bin
+./gnoi_reset \
```

Comment:
Should we add "./" to the other READMEs? It seems to be absent from a few

Reply:
yes, thanks